plugins/webp-uploads/helper.php

```diff
@@ -185,7 +185,7 @@ function webp_uploads_generate_additional_image_source( int $attachment_id, stri
 		return $image;
 	}
 
-	if ( empty( $image['file'] ) ) {
+	if ( ! isset( $image['file'] ) ) {
```

Comment:
Technically:

```suggestion
	if ( ! isset( $image['file'] ) || '' === $image['file'] ) {
```

However, in practice it seems that if `$image` is not a `WP_Error` then really the `file` should always be set. So this entire `if` statement seems it might be worth removing.

Reply:
Updated the check in 375a1ed.

Reply:
I think it might be safer to leave the check in because of the `image_make_intermediate_size` filter applied to the `file` key. Let me know if you feel strongly about removing it!